docs/gen_ref_nav.py

```diff
@@ -24,7 +24,7 @@
         ident = ".".join(parts)
         print("::: " + ident, file=fd)
 
-    mkdocs_gen_files.set_edit_path(full_doc_path, path)
+    mkdocs_gen_files.set_edit_path(full_doc_path, Path("../") / path)
```

Comment:
I did not update that one because we never set edit_uri anywhere, so it might be confusing if they do and then it starts linking wrongly. I will add a note about it tho